test/frontend/jest/sessionDialog.test.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable max-len, quotes */
+'use strict';
+
+const $ = require('jquery');
+const fs = require('fs');
+const path = require('path');
+const sessionTimeoutWarningHtml = fs.readFileSync(path.resolve(__dirname, '../../../frontend/template-partials/views/partials/session-timeout-warning.html'), 'utf8');
+
+jest
+  .dontMock('fs');
+
+describe('sessionDialog', () => {
+  let sessionDialog;
+  let $body;
+  let $html;
+  let options;
+  let originalHTMLDialogElement;
+
+  beforeAll(() => {
```

Comment:
`beforeAll`, can be dangerous because it only does a setup once and then it runs through the tests so it's not fresh but `beforeEach` takes up more resource.  I prefer `beforeEach` just to be on the safe side but I'm sure it's fine in your scenario